src/main/resources/liquibase/crop_changelog/24_3_0.xml

```diff
@@ -0,0 +1,392 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+
+	
+	<changeSet id="v24.3.0-1" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "Inventory" AND cv_id = 1011;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add cvterm class Inventory</comment>
+        <sql dbms="mysql" splitStatements="true">
+            INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (1011, 'Inventory', 'Inventory', null, 0, 0);
+        </sql>
+	</changeSet>	
+	
+	
+	<changeSet id="v24.3.0-2" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "Packing status" AND cv_id = 1010;
```

Comment:
@mchabito This changeset is breaking, what the name should be? 
Packing status or Packing status property? you are using a name for the precondition and to get the Id and another one for the insert

Reply:
Hi @mchabito I found that the right one is Packing status and commited the change